drf_recaptcha/validators.py

```diff
@@ -32,55 +59,70 @@ def testing_validation(self):
                 self.messages["captcha_invalid"], code="captcha_invalid"
             )
 
-    def set_client_ip(self, serializer_field):
+    def _get_secret_key_from_context_or_default(self, serializer_field=None) -> str:
+        if not serializer_field:
+            return self.default_recaptcha_secret_key
+
```

Comment:
```suggestion
    def _get_secret_key_from_context_or_default(self, serializer_field) -> str:
```

Reply:
it would be resolved after https://github.com/llybin/drf-recaptcha/pull/30#discussion_r1241114420